src/components/NavLink.tsx

```diff
@@ -25,7 +25,7 @@ export interface NavLinkProps extends BeamFocusableProps {
   buttonRef?: RefObject<HTMLElement>;
 }
 
-type NavLinkVariant = "side" | "global";
+type NavLinkVariant = "side" | "global" | "normal";
```

Comment:
**thought:** I'm hesitant to add a new `NavLink` variant. These are suppose to be links for our application navigation, like the Global Nav, or the Project Side Nav. `normal` makes me wonder, why not just use a `<a href="..." />` instead?